sites/fast-tooling-examples/app/examples/native-element-1/index.html

```diff
@@ -1,33 +1,21 @@
 <!DOCTYPE html>
 <html lang="en" dir="ltr">
+    <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <%= htmlWebpackPlugin.tags.headTags %>
+        <title><%= htmlWebpackPlugin.options.title %></title>
+        <%= htmlWebpackPlugin.options.manifest %>
+    </head>
 
-<head>
-    <meta charset="utf-8" />
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <%= htmlWebpackPlugin.tags.headTags %>
-    <title><%= htmlWebpackPlugin.options.title %></title>
-    <%= htmlWebpackPlugin.options.manifest %>
-</head>
-
-<body>
-    <div id="root">
-        <input id="text" />
-        <pre id="data-dictionary"></pre>
-    </div>
-    <%= htmlWebpackPlugin
-        .tags
-        .bodyTags
-        .filter(
-            (tag) => 
-                tag.attributes.src.includes('exampleNativeElement1')
-                || !(
-                    tag.attributes.src.startsWith('main')
-                    || tag.attributes.src.startsWith('/example')
-                    || tag.attributes.src.startsWith('/vendors~example')
-                )
-        )
-        .join('')
-    %>
-</body>
-
-</html>
\ No newline at end of file
+    <body>
+        <div id="root">
+            <input id="text" />
+            <pre id="data-dictionary"></pre>
+        </div>
+        <%= htmlWebpackPlugin .tags .bodyTags .filter( (tag) =>
```

Comment:
hmm this looks like it might error now - does this still work??

Reply:
huh... nevermind I  think it's fine. I guess spaces between property accessors is legal

Reply:
Still works fine, I suppose it's treated like js where you can line break at will for chained methods and object properties